public/plugin.tsx

```diff
@@ -131,6 +140,7 @@ export class AssistantPlugin
     }
 
     return {
+      ...dataSourceSetupResult,
```

Comment:
```suggestion
      dataSourceService: dataSourceSetupResult
```
Is there any place that will use `dataSourceSetupResult`, and if so, can we make it a single entry to keep the export interface clean?

Reply:
How about just export a `setDataSourceId` method? For now, this method not been used in any place.

Reply:
Agree that.